### DISCUSSION TITLE
I'm idiot

### DISCUSSION THREAD

**Discussion by sulfur123** [vrcx-team/VRCX#262] [Q&A] [answered]
Can you tell me how to use it in detail

**Comment by Natsumi-sama** [comment, marked as answer]
Extract entire zip contents into a folder then run VRCX.exe

**Reply by sulfur123** [reply]
exe file does not exist

**Reply by Myrkie** [reply]
Probably means you have "show known file types" disabled in windows its disabled by default just run the file called VRCX
its next to the Image

![image](https://user-images.githubusercontent.com/20288698/130053725-5e026b03-4a3f-4016-b419-95b0f923313b.png)

**Reply by sulfur123** [reply]
It's not anywhere
![image](https://user-images.githubusercontent.com/47974364/130057652-dbfa8e86-5f64-4fff-a95f-f3b59fbc810b.png)

**Reply by Myrkie** [reply]
You downloaded the Source code, not the released Binary. please download one of the zip files from here https://github.com/pypy-vrc/VRCX/releases/tag/v2021.08.15 (VRCX_20210815.zip) not (source code)

**Reply by sulfur123** [reply]
Thank you for letting me know. Thanks to you, I've solved it
